projects/1.19/assets/worthy-food/worthy_food/lang/zh_cn.json

```
@@ -0,0 +1,29 @@
+{
+  "item.worthy_food.berry_jam": "甜浆果酱",
+  "item.worthy_food.apple_jam": "苹果酱",
+  "item.worthy_food.toast": "吐司",
+  "item.worthy_food.apple_pie": "苹果派",
+  "item.worthy_food.caramel": "焦糖",
+  "item.worthy_food.cooking_pot": "烹饪锅",
+  "item.worthy_food.fried_egg": "煎蛋",
+  "item.worthy_food.carrot_pie": "胡萝卜派",
+  "item.worthy_food.raw_bacon": "生培根",
+  "item.worthy_food.apple_in_caramel": "焦糖苹果",
+  "item.worthy_food.glowberry_toast": "涂发光浆果酱的吐司",
+  "itemGroup.worthy_food.worthy_food": "有价值的食物",
+  "item.worthy_food.berry_pie": "甜浆果派",
+  "item.worthy_food.apple_juice": "苹果汁",
+  "item.worthy_food.fried_egg_with_bacon": "煎蛋配培根",
+  "item.worthy_food.knife": "小刀",
+  "item.worthy_food.glowberry_jam": "发光浆果酱",
+  "item.worthy_food.flour": "面粉",
+  "item.worthy_food.slice_of_cheese": "起司切片",
```

```suggestion
  "item.worthy_food.slice_of_cheese": "奶酪切片",
```
这个也是我个人一直不好决定的事
奶酪，芝士，起司（现在一般取头两种）